scene/main/scene_tree.cpp

```diff
@@ -1545,10 +1545,16 @@ Ref<SceneTreeTimer> SceneTree::create_timer(double p_delay_sec, bool p_process_a
 Ref<Tween> SceneTree::create_tween() {
 	_THREAD_SAFE_METHOD_
 	Ref<Tween> tween = memnew(Tween(true));
+	tween->parent_tree = this;
```

Comment:
The tree could be a parameter of the constructor.

Reply:
I can certainly add another constructor with a `SceneTree` argument for its parameter and use that! 🙂